lib/companies/companies.ex

```diff
@@ -22,8 +22,11 @@ defmodule Companies.Companies do
 
     (c in Company)
     |> from()
+    |> join(:inner, [c], i in assoc(c, :industry))
+    |> join(:left, [c, _i], j in assoc(c, :jobs))
     |> predicates(params)
-    |> preload([:industry, :jobs])
+    |> distinct([_c, _i, _j], true)
+    |> preload([_c, i, j], industry: i, jobs: j)
```

Comment:
So we can preload these specific joins, we need to add both `industry` and `jobs` joins to our query here. Since there may be no jobs, we go with a left join.